packages/jest-circus/src/legacy_code_todo_rewrite/jest_adapter_init.js

```diff
@@ -233,6 +233,8 @@ const eventHandler = (event: Event) => {
 };
 
 const _addExpectedAssertionErrors = (test: TestEntry) => {
+  if (test.expired) return;
```

Comment:
instead of a new prop here, we could also check if `test.errors > 0` and then not add from `expect`?